Content.Server/Communications/CommunicationsConsoleSystem.cs

```diff
@@ -254,6 +257,9 @@ private void OnSelectAlertLevelMessage(EntityUid uid, CommunicationsConsoleCompo
                 return;
             }
             _chatSystem.DispatchStationAnnouncement(uid, msg, title, colorOverride: comp.AnnouncementColor);
+            // yucky double check if player is valid, but couldn't find a way to ensure it after announcement was sent
+            if (message.Session.AttachedEntity is {Valid: true} player)
+                _adminLogger.Add(LogType.Chat, LogImpact.Low, $"{ToPrettyString(player):player} has sent the following station announcement: {msg}");
```

Comment:
Just check if mob != null

Reply:
I'll mark this as resolved if I get an answer on HowDoICode, but I get the following issue:
"Local variable mob might not be initialized before accessing".
I'm sure there's a way to default mob to initialize as null, but I can't find the syntax for it... mind giving me a hint?

Reply:
Nevermind, I was being dumb